atest/testdata/standard_libraries/builtin/repeat_keyword.robot

```diff
@@ -31,6 +32,12 @@ Invalid Times 2
     [Documentation]    FAIL STARTS: 'notaninteger' cannot be converted to an integer: ValueError:
     Repeat Keyword    Not an integer    No Operation
 
+Repeat Keyword With Time String
+    Repeat Keyword    00:00:03    Keyword Executing For 1 second    00:00:03
+    Repeat Keyword    3 seconds    Keyword Executing For 1 second    3 seconds
+    Repeat Keyword    3 secs    Keyword Executing For 1 second    3 secs
+    Repeat Keyword    3s    Keyword Executing For 1 second    3s
```

Comment:
Repeating keywords for 3 seconds in all these iterations consumes quite a bit time altogether. Please change it to something smaller. If you are only testing time string handling, it ought to be OK to have just 3 milliseconds. If you need to test that the keyword was executed several times, 0.3 seconds ought to be good.